components/application-broker/internal/broker/provision.go

```diff
@@ -3,27 +3,47 @@ package broker
 import (
 	"context"
 	"fmt"
+	"net/http"
 	"sync"
 	"time"
 
-	"net/http"
-
+	"github.com/komkom/go-jsonhash"
```

Comment:
We removed this library from code because of a problem with the license. Details you can find [here](https://github.com/kyma-project/kyma/issues/6142). PR which removes it can be find [here](https://github.com/kyma-project/kyma/pull/6453)

Reply:
Removed. Sorry for the wrong merge ...